src/models/converters/transactions/details.rs

```diff
@@ -51,18 +61,12 @@ impl MultisigTransaction {
                     submitted_at: confirmation.submission_date.timestamp_millis(),
                 })
                 .collect(),
-            signature_info: Some(self.build_signature_info(gas_token)),
-        }
-    }
-
-    fn build_signature_info(&self, gas_token: Option<TokenInfo>) -> SignatureInfo {
-        SignatureInfo {
-            refund_receiver: self.refund_receiver.to_owned(),
+            refund_receiver: self.refund_receiver.as_ref().unwrap_or(&String::from("0x0000000000000000000000000000000000000000")).to_owned(),
+            gas_token: self.gas_token.as_ref().unwrap_or(&String::from("0x0000000000000000000000000000000000000000")).to_owned(),
```

Comment:
very unimportant: could we have the order of the params similar to how it is in the contracts -> nonce, safeTxGas, baseGas, gasPrice, gasToken, refundReceiver